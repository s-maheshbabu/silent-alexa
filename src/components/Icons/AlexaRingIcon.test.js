import React from "react";
import { shallow } from "enzyme";
import AlexaRingIcon from "./AlexaRingIcon";

it("renders AlexaRingIcon without crashing", () => {
  const wrapper = shallow(<AlexaRingIcon />);
  expect(wrapper).toMatchSnapshot();

  wrapper.unmount();
});
