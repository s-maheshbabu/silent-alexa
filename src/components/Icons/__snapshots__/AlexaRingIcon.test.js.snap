// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AlexaRingIcon without crashing 1`] = `
<svg
  viewBox="0 0 24 24"
  width="24"
>
  <path
    d="M0,10.0000489 C0,15.0707816 3.77428289,19.2594477 8.66667972,19.9113334 L8.66667972,17.8962718 C8.66667972,17.3281595 8.30829606,16.8174945 7.76974193,16.636736 C4.94690794,15.688512 2.927648,12.9904434 3.00202582,9.8313279 C3.09245359,5.9853886 6.22532565,2.96152397 10.0722248,3.00037678 C13.9049334,3.03913173 16.9999315,6.15812215 16.9999315,10.0000489 C16.9999315,10.087639 16.9977785,10.1747398 16.9945489,10.2614491 C16.9887748,10.4004189 16.9838815,10.4807669 16.9775203,10.5606256 C16.975563,10.5860707 16.9731163,10.611418 16.9707676,10.6367653 C16.9658743,10.692549 16.9601002,10.7479411 16.9538368,10.8032355 C16.9466926,10.8660654 16.9385698,10.928504 16.9298598,10.9906489 C16.9258473,11.01903 16.9220305,11.0475091 16.9177244,11.0756945 C16.0607158,16.7212922 8.70778325,19.8942068 8.66756051,19.9115291 C9.10355154,19.9694658 9.54815475,20 9.99990213,20 C15.5228467,20 20,15.5229227 20,10.0000489 C20,4.47717519 15.5228467,0 9.99990213,0 C4.47715329,0 0,4.47717519 0,10.0000489 Z"
    fill="#00ACE0"
    transform="translate(2 2)"
  />
</svg>
`;
